src/batou/secrets/encryption.py

```diff
@@ -18,6 +19,15 @@
 """
 
 
+def iter_other_secrets(
+        environment_name: str) -> Generator[pathlib.Path, None, None]:
+    """Iterate over the paths to additional encrypted files."""
+    environment = pathlib.Path("environments") / environment_name
+    for path in environment.iterdir():
+        if path.name not in ('environment.cfg', 'secrets.cfg'):
```

Comment:
Yeah, this is too coarse. We need to keep using secrets-name.ext because there may be other manual files (like provision.sh and provision.nix and even further things). I don't think a sub-directory is needed, though.

Reply:
I am fixing this in #266. For this change I had to redo the whole migration.